configs/default/DIAT-MAMBAG4.config

```
@@ -21,11 +21,13 @@ resource SERIAL_TX 1 A09
 resource SERIAL_TX 2 B03
 resource SERIAL_TX 3 B10
 resource SERIAL_TX 4 C10
-resource SERIAL_TX 11 B09
+resource SERIAL_TX 11 B10
 resource SERIAL_RX 1 A10
 resource SERIAL_RX 2 B04
 resource SERIAL_RX 3 B11
 resource SERIAL_RX 4 C11
+resource SERIAL_RX 11 B11
+resource SOFTSERIAL_TX 1 B09
```

Replace with LPUART (shared with UART5)
```suggestion
resource LPUART1_TX_PIN B11
```